### DISCUSSION TITLE
Does Detours support ARM64EC architecture?

### DISCUSSION THREAD

**Discussion by js1016** [microsoft/Detours#320] [Q&A] [answered]
I am exploring the use of Microsoft's Detours library for intercepting functions on different architectures and was wondering if Detours supports the ARM64EC architecture. ARM64EC (ARM64 Emulation Compatible) is a newer architecture introduced by Microsoft to allow x64 code modules to run on ARM64 systems, providing a bridge for compatibility in Windows on ARM devices.

Given Detours' support for ARM64 and other architectures, can you confirm whether it also supports ARM64EC? Are there any known issues or limitations when using Detours with ARM64EC, especially when dealing with mixed ARM64 and x64 code modules?

Any insights or guidance would be greatly appreciated!

**Comment by BissetJ** [comment, marked as answer]
FYI, ARM64EC support was just added in #338.
